### ISSUE TITLE
5.194 - "RequestTracingService was started at level -2 but it has a run level of 10"

### ISSUE THREAD

**Issue by jportner-hw** [payara/Payara#4459]
We are trying to upgrade to payara 5.194 from 5.183

The following error is thrown when using javax.ws.rs.client.Client to connect to a WireMock localhost server within a unit test.
We are using the payara-embedded-all dependency.

```
java.lang.IllegalStateException: Service fish.payara.nucleus.requesttracing.RequestTracingService was started at level -2 but it has a run level of 10.  The full descriptor is SystemDescriptor(
	implementation=fish.payara.nucleus.requesttracing.RequestTracingService
	name=requesttracing-service
	contracts={fish.payara.nucleus.requesttracing.RequestTracingService,org.jvnet.hk2.config.ConfigListener,fish.payara.monitoring.collect.MonitoringDataSource}
	scope=org.glassfish.hk2.runlevel.RunLevel
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=runLevelValue={10}
	rank=0
	loader=org.glassfish.hk2.bootstrap.impl.Hk2LoaderPopulatorPostProcessor$1@7b2bf745
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=1243
	locatorId=1
	identityHashCode=254642444
	reified=true)
	at org.glassfish.hk2.runlevel.internal.AsyncRunLevelContext.validate(AsyncRunLevelContext.java:422)
```

Is there a way to disable the RequestTracingService?
Or any ideas why it is failing to start at the correct run level?

## Environment ##

- **Payara Version**: 5.194
- **Edition**: Full
- **JDK Version**:  Zulu 8
- **Operating System**: Mac/Linux
- **Database**:  MySQL

**Comment by dmatej** [comment]
Hi, this problem will be fixed in Payara 5.194.2 for customers with paid support, which should be released in next few days, and in Payara 5.201, which should be released in february.

The problem was caused by unwanted interaction of Jersey Client with Embedded Payara on the same classpath, so it detected a filter inside embedded jar which depended on some internal services which were not started yet.

**Comment by rdebusscher** [comment]
Hi Josh,

As mentioned on the Zendesk ticket, we will notify you when the customer release is available at the end of the week.

Regards
Rudy

**Comment by rdebusscher** [comment]
Hi Josh,

This should be fixed as part of the customer release 5.194.2 which is released now (and scheduled to be included ion community release 5.201)

Regards
Rudy
